ast/src/main/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitor.java

```diff
@@ -159,7 +160,10 @@ public void visitPrecisionDeclaration(PrecisionDeclaration precisionDeclaration)
   public void visitDeclarationStmt(DeclarationStmt declarationStmt) {
     out.append(indent());
```

Comment:
Do not do this indent if we are in a for statement header.

This will avoid a space after the opening paren of "for(" in "for(int i"

Reply:
Fixed.